onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -204,7 +204,10 @@ void NchwcGlobalPoolOpSchemaGenerator(OpSchema& schema) {
 void ValidateTypeAndShapeForScaleAndZP(ONNX_NAMESPACE::InferenceContext& ctx, int index, ::google::protobuf::int32 expectedType, bool isScalar, int expectedTensorSize = 0) {
   if (ctx.getNumInputs() > static_cast<size_t>(index)) {
     auto data_type = ctx.getInputType(index);
-    if (nullptr == data_type || data_type->value_case() != ONNX_NAMESPACE::TypeProto::kTensorType ||
+    if (nullptr == data_type) {
```

Comment:
This change is for fixing a potential null pointer error.

Reply:
What is the error? If data_type is nullptr it won't evaluate the rest of that statement.

Reply:
While printing the error message